bin/icapps_generator.dart

```diff
@@ -3,31 +3,30 @@ import 'dart:io';
 import 'package:path/path.dart';
 
 import 'src/file_creator_helper.dart';
-import 'src/flutter_helper.dart';
 import 'src/params.dart';
 
 var screenName = '';
-Params params;
+late final Params params;
 const NO_NAV_ARG = '--no-nav';
-const NO_KIWI_ARG = '--no-kiwi';
+const NO_INJECTABLE_ARG = '--no-di';
```

Comment:
your idea of moving to --no-di is indeed more generic. But I think we should use an other constant name. NO_DI_ARG for example